tests/schema/tests.py

```diff
@@ -2758,6 +2758,46 @@ def test_alter_field_default_doesnt_perform_queries(self):
         with connection.schema_editor() as editor, self.assertNumQueries(0):
             editor.alter_field(AuthorWithDefaultHeight, old_field, new_field, strict=True)
 
+    @unittest.skipIf(connection.vendor == 'sqlite', 'SQLite naively remakes the table on field alteration.')
```

Comment:
This should work on SQLite as well

https://github.com/django/django/blob/292b3be698ef58aff9c215d62a444f66ead578c3/django/db/backends/sqlite3/schema.py#L101-L103

Reply:
Yes it does! Thanks 👍